### PR TITLE
AWS S3 SDK created & Non-API request in NetworkService added.

### PR DESCRIPTION
**Pull request by rado196**
1) AWS S3 SDK added with generating signed URL for `putObject` action and then uploads the file to amazon bucket.
2) Added a new function that will allow us to send the non-API requests.

### CODE REVIEW

File: packages/services/NetworkService/index.ts [hunk 1]
```diff
@@ -68,64 +68,34 @@ class NetworkService {
...
+        } else {
+          accumulator.push(`${key}=${item}`);
         }
-      } else {
-        accumulator += `${key}=${item}&`;
-      }
-      return accumulator;
-    }, '');
+
+        return accumulator;
```
Comment (rubenaprikyan) [inline packages/services/NetworkService/index.ts:86]: This is for removing `&` postfix?

File: packages/services/NetworkService/index.ts [hunk 2]
```diff
@@ -164,6 +134,62 @@ class NetworkService {
         })
         .catch((err: any) => reject(err));
     });
+  };
```
Comment (rubenaprikyan) [inline packages/services/NetworkService/index.ts:137]: Why ?

File: sdks/core-api-sdk/src/sdks/AwsS3SDK/index.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import { HttpRequestMethods } from '@ultras/services/NetworkService/types';
+import CoreApiBaseSDK, { Mode } from '../CoreApiBaseSDK';
+import { DynamicQueryParam } from '../types';
+import {
+  SigningUrlParamsInterface,
+  UploadViaSignedUrlParamsInterface,
+  UploadParamsInterface,
+} from './types';
+
+export class AwsS3SDK extends CoreApiBaseSDK {
```
Comment (rubenaprikyan) [inline sdks/core-api-sdk/src/sdks/AwsS3SDK/index.ts:10]: Please change the name of this, so mention this is `Ultras` or `Custom`, there is an sdk for official s3 service

File: sdks/core-api-sdk/src/sdks/AwsS3SDK/index.ts [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    super(mode, 's3');
+  }
+
+  public getSignedUrl(params: SigningUrlParamsInterface) {
+    return this.api?.makeAPIGetRequest('signed-url', {
+      query_params: params as DynamicQueryParam,
+    });
+  }
+
+  public uploadViaSignedUrl(params: UploadViaSignedUrlParamsInterface) {
```
Comment (rubenaprikyan) [inline sdks/core-api-sdk/src/sdks/AwsS3SDK/index.ts:21]: Please write a brief descriptions

**Review by rubenaprikyan — CHANGES_REQUESTED** [review]

File: packages/services/NetworkService/index.ts (hunk 1 above)
Comment (rado196) [reply packages/services/NetworkService/index.ts:86]: yes

File: sdks/core-api-sdk/src/sdks/AwsS3SDK/index.ts (hunk 3 above)
Comment (rado196) [reply sdks/core-api-sdk/src/sdks/AwsS3SDK/index.ts:10]: did you mean `UltrasS3SDK`? we can use `FileUploadS3SDK`

File: packages/services/NetworkService/index.ts (hunk 2 above)
Comment (rado196) [reply packages/services/NetworkService/index.ts:137]: I think more than 3 lines of code writing inline is not human readable.

**Review by rubenaprikyan — COMMENTED** [review]
lgtm!

**Pull request merged**